src/components/TagCloud.js

```diff
@@ -7,10 +7,13 @@ export class TagCloudValue {
   label: string;
   /** The value for this item. */
   value: number;
+  /** Whether the tag should be clickable */
+  noLink: boolean;
 
-  constructor(label: string, value: number) {
+  constructor(label: string, value: number, noLink: boolean) {
```

Comment:
I would add an explicit default value of `false` to the noLink parameter of the constructor. This will help people know that they don't have to pass this (in case they're not doing so now) and it'll make it explicit that not doing to will create a clickable tag item.